zkevm-circuits/src/tx_circuit.rs

```diff
@@ -772,16 +820,49 @@ impl<F: Field> SubCircuitConfig<F> for TxCircuitConfig<F> {
                     meta.query_advice(calldata_gas_cost_acc, Rotation::next()),
                     meta.query_advice(calldata_gas_cost_acc, Rotation::cur()) + gas_cost_next,
                 );
+                cb.require_equal(
+                    "calldata_rlc' = calldata_rlc * r + byte'",
+                    meta.query_advice(calldata_rlc, Rotation::next()),
+                    meta.query_advice(calldata_rlc, Rotation::cur()) * challenges.keccak_input()
+                        + meta.query_advice(tx_table.value, Rotation::next()),
```

Comment:
This calculation is insecure because `tx_table.value` is in SecondPhase.

Reply:
fixed in the https://github.com/scroll-tech/zkevm-circuits/pull/735. I copied the tx_table.value of calldata part to a newly allocated 1st phase column.